pkg/controller/direct/workstations/config_mappings.go

```diff
@@ -26,15 +26,32 @@ func WorkstationConfigObservedState_FromProto(mapCtx *direct.MapContext, in *pb.
 		return nil
 	}
 	out := &krm.WorkstationConfigObservedState{}
-	// MISSING: Name
-	// MISSING: Uid
-	// MISSING: Reconciling
+	out.UID = direct.LazyPtr(in.GetUid())
 	out.CreateTime = direct.StringTimestamp_FromProto(mapCtx, in.GetCreateTime())
 	out.UpdateTime = direct.StringTimestamp_FromProto(mapCtx, in.GetUpdateTime())
 	out.DeleteTime = direct.StringTimestamp_FromProto(mapCtx, in.GetDeleteTime())
 	out.Etag = direct.LazyPtr(in.GetEtag())
+	out.Host = WorkstationConfig_HostObservedState_FromProto(mapCtx, in.Host)
 	out.Degraded = direct.LazyPtr(in.GetDegraded())
-	// MISSING: Conditions
+	out.GCPConditions = WorkstationGCPConditions_FromProto_Alpha(mapCtx, in.Conditions)
```

Comment:
Just out of curiosity, why is this called `_alpha`?

Reply:
There is a difference in type between v1beta1 / v1alpha1. I will remove the `_Alpha` functions once I promote this resource (and other Workstation resources) to v1beta1.